lua/autorun/photon/cl_emv_meta.lua

```diff
@@ -1006,4 +1006,385 @@ function EMVU:MakeEMV( emv, name )
 	emv.PhotonFinishedInit = true
 end
 
+local NextDoppler = 0
+local thirdPersonVolume = 0.6
+local interiorVolume = 0.2
+local fadeDist = 4500
+local updateRate = 0.1
+
+hook.Add("Think", "Photon.ELS_SirenDoppler", function()
+	if NextDoppler < CurTime() then
+		for _,v in ipairs(ents.FindByClass("prop_vehicle_jeep")) do
+			if v.Siren or v.Siren2 or v.ManualSiren or v.Horn then
+				local ply = LocalPlayer()
+				local pos = ply:GetPos()
+				local spos = v:GetPos()
+				local doppler = ((pos:Distance(spos+ply:GetVelocity())-pos:Distance(spos+v:GetVelocity()))/200)
+				if IsValid(ply:GetVehicle()) then
+					if ply:GetVehicle():GetParent() == v then
```

Comment:
We can check the player's vehicle outside the loop, plyVeh: false|Vehicle. I'm assuming this is for seats?